docs/release-notes/v11.4.0.md

```diff
@@ -0,0 +1,97 @@
+---
+id: v11.4.0
+title: Version v11.4.0 Release Notes
+sidebar_label: v11.4.0
+image: "img/release-note-link-preview.png"
+---
+
+_July 06th, 2023_
+
+## Console
+
+### New Features
+
+#### What's new section
+
+A big welcome to [What's new section](/development_suite/overview-dev-suite#whats-new.md), that will keep up to date with the latest product news!  
+You can access this Console section by clicking on the question mark icon located in the top bar of the Console interface.
+In case there is unread news, a notification symbol will appear so that you do not miss the latest updates that can improve your experience on Mia-Platform Console.
```

Comment:
```suggestion
A big welcome to the [What's new section](/development_suite/overview-dev-suite#whats-new.md), where you can stay up to date with the latest product news!  
You can access this section by clicking on the question mark icon located in the top bar of the Console interface.
In case there is any unread news, a notification symbol will appear, so that you do not miss the latest updates that can improve your experience on Mia-Platform Console!
```